core/src/test/java/org/apache/brooklyn/location/byon/ByonLocationResolverTest.java

```diff
@@ -345,7 +345,7 @@ public void testWindowsMachines(String osFamily) throws Exception {
         String spec = "byon";
         Map<String, ?> flags = ImmutableMap.of(
                 "hosts", ImmutableList.of("1.1.1.1", "2.2.2.2"),
-                "osfamily", osFamily
+                "osFamily", "windows"
```

Comment:
I am doing exactly this.
I've changed it from osfamily to osFamily.
Notice that this was only in ByonLocationResolver. Can you check my PR comment on top. What do you think for moving this parameter in AbstractLocation